app/controllers/proposals_controller.rb

```diff
@@ -2,21 +2,27 @@ class ProposalsController < ApplicationController
   before_action :set_proposal
 
   def index
-    @proposals = current_user.proposals
+    if user_signed_in? && current_user.pilot
+      @proposals = Proposal.all
+    elsif user_signed_in? && !current_user.pilot
+      @proposals = current_user.proposals
```

Comment:
pilots should only see the proposals they made
@proposals = current_user.proposals ?